tests/templates/ingress.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         app: microbot
     spec:
       containers:
-        - image: cdkbot/microbot-$ARCH
+        - image: cdkbot/microbot-amd64
```

Comment:
Please do not commit this.

Reply:
It always happens at local execution. Removing it.